viz_scripts/scaffolding.py

```diff
@@ -108,7 +108,12 @@ def expand_userinputs(labeled_ct):
 unique_users = lambda df: len(df.user_id.unique()) if "user_id" in df.columns else 0
 trip_label_count = lambda s, df: len(df[s].dropna()) if s in df.columns else 0
 
-def load_viz_notebook_data(year, month, program, study_type, has_dynamic_labels, dynamic_labels_url, dic_re, dic_pur=None, include_test_users=False):
+# Function to map: Selected an "old_mode" col from expanded_ct and 
+# map to the value of dic_translations into "new mode" col of expanded_ct
+def update_expanded_ct(expanded_ct, new_mode, old_mode, dic_mapping):
+    expanded_ct[new_mode] = expanded_ct[old_mode].map(dic_mapping)
+
```

Comment:
creating a new function for one line of code is overkill. It requires a new call stack AND a new context jump from those reviewing the code.

Reply:
I saw three instances where this mapping was happening [I thought this could be expanded in future], so I wanted to create a new function for it. I've reverted it to use in-place mapping functionality rather than calling a new function.